### DISCUSSION TITLE
ModuleNotFoundError: No module named 'vanna.remote'; 'vanna' is not a package

### DISCUSSION THREAD

**Discussion by guyzilb** [vanna-ai/vanna#296] [Q&A] [answered]
import vanna
from vanna.remote import VannaDefault
vn = VannaDefault(model='chinook', api_key=vanna.get_api_key('my-email@example.com'))
vn.connect_to_sqlite('https://vanna.ai/Chinook.sqlite')
vn.ask("What are the top 10 albums by sales?")
from vanna.flask import VannaFlaskApp
VannaFlaskApp(vn).run() 

 What I miss here ?

**Comment by zainhoda** [comment, marked as answer]
Typically when we see this it's because your file is named `vanna.py` so Python is trying to import the current file you have open. Try changing your filename from `vanna.py` to anything else (for example `vanna-app.py`)
